src/localsocketcontroller.h

```diff
@@ -5,13 +5,17 @@
 #ifndef LOCALSOCKETCONTROLLER_H
 #define LOCALSOCKETCONTROLLER_H
 
-#include <QHostAddress>
 #include <QLocalSocket>
 #include <QTimer>
 #include <functional>
 
 #include "controllerimpl.h"
 
+// For messages that are expected to generate a synchronous response, this
+// defines the default time that we will wait before assuming an error in
+// the daemon has occured.
+constexpr int CONNECTION_RESPONSE_TIMEOUT_MSEC = 500;
```

Comment:
Consider moving this into the class to avoid affecting the global name space.

Reply:
Done!